cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Gov/DRepActivity.hs

```diff
@@ -192,44 +192,25 @@ activityChangeProposalTest execConfig epochStateView configurationFile socketPat
   (EpochNo epochAfterProp) <- getCurrentEpochNo epochStateView
   H.note_ $ "Epoch after \"" <> prefix <> "\" prop: " <> show epochAfterProp
 
-  waitAndCheck epochAfterProp
+  mResult <- watchEpochStateView epochStateView isSuccess minWait maxWait
+
+  void $ H.evalMaybe mResult
```

Comment:
```suggestion
  H.nothingFailM $ watchEpochStateView epochStateView isSuccess minWait maxWait
```

Reply:
Fixed in: https://github.com/IntersectMBO/cardano-node/commit/a3b7e39d18dcbe5e345fc2de8e721927e89abb86